model/src/main/proto/spine_examples/chatspn/command/commands.proto

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Let's keep the Proto files with commands and events under `user`:
```
-> spine_examples
    -> chatspn
        -> identifiers.proto (once we need it)
        -> user
            -> commands.proto
            -> events.proto 
            -> user.proto 
            -> (other .proto files, such as `values.proto`, or something more specific)
```

`commands.proto`, `events.proto`, and `user.proto` will have Proto package equal to `spine_examples.chatspn.user`. However, their Java packages will differ:

* Commands will be generated into Java package ending with `.user.command`. 
* Events will go to Java package ending with `.user.event`. 
* Types defined in `user.proto` will go directly to the package ending with `.user`.

@Artem-Semenov-dev FYI.